client/app/dbaas/logs/detail/roles/add/logs-role-add.html

```diff
@@ -0,0 +1,75 @@
+<div class="oui-modal oui-modal_shadow">
```

Comment:
You should use the components in modal.component.js

Reply:
I tried using the component, but it still renders cui-modal, which means i will have to overwrite some properties on oui-modal__body like background-color manually. Is it okay ?

Or is there anyway to render oui-modal component directly using ControlHelper ?

Reply:
And also if i do change it will be a hack until the oui modal works properly

Reply:
Ah I didn't know the modal yo uwere using is not the same visually as cui-modal.  Sorry